web/services/installation.go

```diff
@@ -0,0 +1,39 @@
+package services
+
+import (
+	"github.com/google/uuid"
+	"github.com/trento-project/trento/web/entities"
+	"gorm.io/gorm"
+)
+
+//go:generate mockery --name=Installer --inpackage --filename=installer_mock.go
+
+type Installer interface {
+	InitializeIdentifier() error
+}
+
+type installer struct {
+	db *gorm.DB
+}
+
+func NewInstallerService(db *gorm.DB) Installer {
+	return &installer{db: db}
+}
+
+func (i *installer) InitializeIdentifier() error {
```

Comment:
I think it would be nice to return the identifier in this method, just to print it or whatever reason

Reply:
makes sense :+1: